extensions/amp-story/1.0/amp-story-reaction-quiz.css

```diff
@@ -0,0 +1,447 @@
+     --option-4-percentage: 0%;
+     font-family: 'Poppins-bold', sans-serif !important;
+     background: var(--reaction-accent-color) !important;
+     border-radius: 32px !important;
+ }
+ 
+ /* TODO: remove bg on no-prompt attribute here */
+ 
+ .i-amphtml-story-reaction-quiz-prompt-container {
+     display: flex !important;
+     justify-items: center !important;
+     padding: 20px !important;
+     color: var(--reaction-prompt-text-color) !important;
+ }
+ 
+ .i-amphtml-story-reaction-quiz-prompt {
+     margin: 0px !important;
+     overflow : hidden !important;
+     text-overflow: ellipsis !important;
+     display: -webkit-box !important;
+     -webkit-line-clamp: 4 !important;
+     -webkit-box-orient: vertical !important;
+     visibility: visible !important;
+ }
+ 
+ h1.i-amphtml-story-reaction-quiz-prompt {
+     font-size: 28px !important;
+     line-height: 36px !important;
+     max-height: 144px !important;
+ }
+ 
+ h2.i-amphtml-story-reaction-quiz-prompt {
+     font-size: 22px !important;
+     line-height: 28px !important;
+     max-height: 112px !important;
+ }
+ 
+ h3.i-amphtml-story-reaction-quiz-prompt {
+     font-size: 18px !important;
+     line-height: 24px !important;
+     max-height: 96px !important;
+ }
+ 
+ .i-amphtml-story-reaction-quiz-option-container {
+     display: flex !important;
+     flex-direction: column;
+     background-color: var(--reaction-theme-background) !important;
+     border-radius: var(--reaction-chip-radius) !important;
+     padding: 8px 8px 0px 8px !important;
+ }
+ 
+ .i-amphtml-story-reaction-quiz-option {
+     position: relative !important;
+     display: flex !important;
+     justify-items: start !important;
+     align-items: center !important;
+     border-radius: var(--reaction-chip-radius) !important;
+     padding: 8px 16px 8px 8px !important;
+     background-color: var(--reaction-theme-background) !important;
+     margin-bottom: 8px !important;
+     color: var(--reaction-option-text-color) !important;
+     border: solid 1px var(--reaction-theme-border) !important;
+     box-shadow: var(--reaction-chip-shadow) !important;
+     -webkit-box-shadow: var(--reaction-chip-shadow) !important;
+     -moz-box-shadow: var(--reaction-chip-shadow) !important;
+     font-size: 16px !important;
+     line-height: 20px !important;
+     overflow: hidden !important;
+     z-index: 0 !important;
+ }
+ 
+ [dir=rtl] .i-amphtml-story-reaction-quiz-option {
+     padding: 8px 8px 8px 16px !important;
+ }
+ 
+ /* Truncate option text and add an ellipsis after 2 lines. */
+ .i-amphtml-story-reaction-quiz-option-text {
+     max-height: 40px !important;
+     overflow : hidden !important;
+     text-overflow: ellipsis !important;
+     display: -webkit-box !important;
+     -webkit-line-clamp: 2 !important;
+     -webkit-box-orient: vertical !important;
+     visibility: visible !important;
+ }
+ 
+ .i-amphtml-story-reaction-post-selection .i-amphtml-story-reaction-option-selected {
```

Comment:
Some classes are prefixed with -reaction- and others with -reaction-quiz- since the abstract class adds those classes to the reaction-option automatically. Things like adding the option-selected and post-selection state get handled by the abstract class as it corresponds to all multiple-choice quizzes and polls.

Reply:
👍